internal/jstest/jstest.go

```diff
@@ -61,6 +64,9 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 			}
 		}
 
+		// to avoid unexpected results
+		testName = strings.TrimPrefix(testName, prefix+"/")
```

Comment:
In general, that should be done using `path/filepath` package, not `path` or `strings`.

But I wonder if we could update mongo.yml instead?

Reply:
The only other solution I could think of is to chdir.

PTAL.

Reply:
No, I'm not sure what is the best solution, but `chdir` is no-go